### PR TITLE
debug for failures on jenkins

### PR DESCRIPTION
**Pull request by gabemontero**
@bparees @csrwng more extended test debug

### CODE REVIEW

**Comment by gabemontero** [comment]
[testonlyextended][extended:core(builds)]

File: test/extended/builds/s2i_dropcaps.go [hunk 1]
```diff
@@ -36,6 +36,15 @@ var _ = g.Describe("[builds][Slow] Capabilities should be dropped for s2i builde
 			g.By("starting the rootable-ruby build with --wait flag")
 			err = oc.Run("start-build").Args("rootable-ruby", fmt.Sprintf("--from-dir=%s", s2ibuilderFixture),
 				"--wait").Execute()
+			// debug for failures on jenkins
+			if err != nil {
+				bldOuput, derr := oc.Run("logs").Args("-f", "bc/rootable-ruby").Output()
+				if derr == nil {
+					g.By(fmt.Sprintf("\n\n  build logs : %s\n\n", bldOuput))
```
Comment (bparees) [inline test/extended/builds/s2i_dropcaps.go]: switch to printing to ginkgowriter.  (same for the rest)

**Comment by gabemontero** [comment]
GinkgoWriter conversion pushed

**Comment by gabemontero** [comment]
[testonlyextended][extended:core(builds)]

**Comment by bparees** [comment]
lgtm, needs @smarterclayton sign off.  extended tests only, no production code changes.

**Comment by smarterclayton** [comment]
Approved

On Apr 19, 2016, at 10:31 PM, Ben Parees <notifications@github.com> wrote:

lgtm, needs @smarterclayton <https://github.com/smarterclayton> sign off.
extended tests only, no production code changes.

—
You are receiving this because you were mentioned.
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/8564#issuecomment-212217829>

**Comment by bparees** [comment]
[merge]

Ben Parees | OpenShift

**Comment by gabemontero** [comment]
@bparees @csrwng FYI - the s2i_dropcaps.go failure I was debugging with this PR failed with the debug.  It was another instance of the push to the registry failing because the disk quota was exceeded.

If there is a "registry leak" of some sort and our tests should not be filling up 25 Gig, I'm not sure offhand how to debug that.  Otherwise, per the email thread, the evidence is mounting for having any jenkins job that can run builds extended tests get a larger AWS volume.

Build log:

F0420 02:55:24.778609       1 builder.go:204] Error: build error: Failed to push image: Received unexpected HTTP status: 500 Internal Server Error

Registry log:

time="2016-04-20T02:55:24.586114462Z" level=error msg="response completed with error" err.code=UNKNOWN err.detail="filesystem: write /registry/docker/registry/v2/repositories/extended-test-build-s2i-dropcaps-jxchp-m5xe8/rootable-ruby/_uploads/6eaf719c-816b-448a-bf07-42da8fd6fa1e/data: disk quota exceeded" err.message="unknown error" go.version=go1.6 http.request.host="172.30.175.173:5000" http.request.id=919f6e23-34e4-4176-a952-8f7b9e60f30a http.request.method=PATCH http.request.remoteaddr="172.18.0.107:56194" http.request.uri="/v2/extended-test-build-s2i-dropcaps-jxchp-m5xe8/rootable-ruby/blobs/uploads/6eaf719c-816b-448a-bf07-42da8fd6fa1e?_state=APiF7YHxMOhlmCCjzxSIcec2t4qO6XGiKTfjPHhI9i57Ik5hbWUiOiJleHRlbmRlZC10ZXN0LWJ1aWxkLXMyaS1kcm9wY2Fwcy1qeGNocC1tNXhlOC9yb290YWJsZS1ydWJ5IiwiVVVJRCI6IjZlYWY3MTljLTgxNmItNDQ4YS1iZjA3LTQyZGE4ZmQ2ZmExZSIsIk9mZnNldCI6MCwiU3RhcnRlZEF0IjoiMjAxNi0wNC0yMFQwMjo1NToxNi41MzA3MDUzNzhaIn0%3D" http.request.useragent="docker/1.9.1 go/go1.4.2 kernel/3.10.0-229.7.2.el7.x86_64 os/linux arch/amd64" http.response.contenttype="application/json; charset=utf-8" http.response.duration=7.82809092s http.response.status=500 http.response.written=292 instance.id=5c885730-2631-44c5-a4e8-cf2a43c7f964 vars.name="extended-test-build-s2i-dropcaps-jxchp-m5xe8/rootable-ruby" vars.uuid=6eaf719c-816b-448a-bf07-42da8fd6fa1e

**Comment by smarterclayton** [comment]
Can we test prune here and use that?

**Pull request merged**